### ISSUE TITLE
Unsubscribe after Upgrade

### ISSUE THREAD

**Issue by memoo2u** [justingit/dada-mail#578]
Hi,

I have been using Dada Mail since version 7. The Unsubscribe link was very useful on a private list.

Since upgrading the Unsubscribe link just leads to a page with a backlink to the website.

How can I make this work? How can people unsubscribe?

I still have the below in my Email Template:

```
  <!-- tmpl_var subscriber.email --> 
 </strong>
</p>


<p style="font:.7em/.9em Helvetica,Verdana,'Sans-serif'">

<!-- tmpl_if PROFILE_ENABLED --> 
<strong>
 <a href="<!-- tmpl_var PROGRAM_URL -->/profile_login/<!-- tmpl_var subscriber.email_name -->/<!-- tmpl_var subscriber.email_domain -->/"></a>
 </strong>
```

```
<!-- /tmpl_if --> 

<strong>
 <a href="<!-- tmpl_var list_unsubscribe_link -->">
  Unsubscribe Automatically &#187;
 </a> 
</strong>

<!--/signature-->`
```

**Comment by memoo2u** [comment]
OK, since nobody knows about fixing this bug. Is there a link or a doc which explains how to do a clean install with a fresh database and still be able to use the same ListName and Email address? I don't want the list to have to write emails to a new email address.

**Comment by justingit** [comment]
> OK, since nobody knows about fixing this bug.

Yikes, give me a second to get to it ;) 

I'm afraid I may need more information to get to the bottom of the problem. Can you tell me: 

* The version of Dada Mail you're currently using
* Any steps needed to recreate the problem, starting from creating a new list
* Any error logs that are specific to the date/time this problem takes place? 

If I can't recreate the problem,  I won't be able to solve it. Trying just to make an unsubscription requst on a private list doesn't seem to show a problem - I get the confirmation screen that my unsubcription request is successful.

**Comment by memoo2u** [comment]
This was bugging me so, I decided to do some file comparisons from version 7.2.0 to 9.2.1. For some reason, the line `<!-- tmpl_var content --> ` is removed from the template file during the upgrade. It was removed during both attempts. the strange part is, the green bar on the "Your Mailing List Template" page reads "tag found in default template". No error message was shown. Anyway, I added the tag back in and I am good to go!
